symphony-bdk-core/src/test/java/com/symphony/bdk/core/auth/impl/AbstractBotAuthenticatorTest.java

```diff
@@ -34,6 +35,7 @@ protected String authenticateAndGetToken(ApiClient client) throws ApiException {
     }
 
     @Override
+    @Nonnull
```

Comment:
Not sure is is relevant here

Reply:
Indeed there is warning because @nonnull is not set for implementations but for tests we have dummy implementations
Not sure if there is way to not return null here but I prefer to avoid warnings in the production code and keep them in the test code

Reply:
I used Mockito to prevent that